client/config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml

```diff
@@ -18,35 +18,133 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
-    - description: Indicates if a snapshot is ready to be used to restore a volume.
+    - description: Indicates if the snapshot is ready to be used to restore a volume.
       jsonPath: .status.readyToUse
       name: ReadyToUse
       type: boolean
-    - description: Name of the source PVC from where a dynamically taken snapshot
-        will be created.
+    - description: If a new snapshot needs to be created, this contains the name of the source PVC from which this snapshot was (or will be) created.
```

Comment:
I actually prefer the original description better, its shorter and clear enough ;)

Reply:
This is changed to address Saad's comments.